books/management/commands/addepub.py

```diff
@@ -39,8 +39,6 @@ def validate_and_add(path, filenames):
         if os.path.splitext(path)[1] == '.epub':
             filename = os.path.abspath(path)
             if filename not in filenames:
-                if not models.Book.objects.filter(
-                        original_path=filename).exists():
```

Comment:
This basically reverts some of the changes introduced on 5c7eb909fe108902343226a1bd7a1a5cf33e4515 - `validate_and_add` is used by the `resync` command, as it needs to return all the "candidate" epubs (even if they seem to be on the DB) for properly comparing and resyncing.

I'm unsure if you include the extra check for other purposes - if that is the case, we can probably add a parameter to the function (`include_on_db=False`) or something similar to avoid duplicating the code.